spec/javascript/utils/form/validations.test.js

```diff
@@ -0,0 +1,56 @@
+import validate from 'utils/form/validations'
+
+// Validations will return this message if failed, null if passed
+const FAILED_VALIDATION = 'failed validation'
```

Comment:
could we update this name to `FAILED_VALIDATION_MSG`? that way it's easier to read that it's not a boolean validation result or an object or field that failed validation